README.md

```diff
@@ -1163,7 +1205,9 @@ rm -rf keys.json keys.json.bck signatures
         },
         "content": "liPEEBCy4aRWs0//mtrMjCD5MBbEQNH2DF4JHY6ADjxPY11f4HwcW/ZJfVonqz6M8/rrIvli1DUw/fGpdEF+k1gd4taYON1wy9P5R3/BPuAIVnxRZQkAxCA8bg4ZY7ZNQrMWP0ZIHxTMAAAAAAAAAAAAAAAAAAAAAMRAtCx79HokXAELQRbiEoE9YVPLfx4Zdh9fC93QO4X4e60HXseQUdVFtbuQBQiz2yqHBuoQyMQVsu2fBPreNihifA=="
       },
-      "requestID": "3c6e0e19-63b6-4d42-b316-3f46481f14cc"
+      "requestID": "3c6e0e19-63b6-4d42-b316-3f46481f14cc",
+      "responseSignatureVerified": true,
+      "responseChainVerified": true
```

Comment:
if the responseSignatureVerified fails, or it is false, is the body fields the same as here? or is there like another response?

Reply:
the fields would be present but with the value would be set to `false` and the status code if the HTTP response would be `502` - Bad Gateway. Note that it is possible for the Signature to be valid but the response UPP had the type signed (not chained) and therefore the response chain can not be verified. in this example the response would contain 
```
      "responseSignatureVerified": true,
      "responseChainVerified": false
```

Reply:
Thanks for the explanation. Maybe you add some hints about this field in the documentation - if not there already -

Reply:
there is some documentation about this at a different part of the readme. i also added a few more sentences now https://github.com/ubirch/ubirch-client-go/pull/117/commits/2873f94c76d72c6c4f0c55108a8f0711a418f8fe

Reply:
thank you!